### PR TITLE
Fix StatusBar colors

### PR DESCRIPTION
**Pull request by piazzatron**
See: https://github.com/AudiusProject/audius-client/pull/194

Additionally, I now have the StatusBar invisible now during iOS splashscreen (since we don't necessarily know the correct color then, and the splash screen looks better without a StatusBar).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "react-native-static-server": "^0.4.2",
     "react-native-version-number": "^0.3.6",
     "react-native-video": "file:react-native-video-5.1.0-alpha2.tgz",
-    "react-native-webview": "^7.4.3",
+    "react-native-webview": "^11.0.2",
```
Comment (raymondjacobson) [inline package.json:52]: damn 3 majors??!? wewwwwww. hopefully this fixes a bunch of stuff. excited to get this on testflight.

File: src/components/web/WebApp.tsx [hunk 2]
```diff
@@ -445,6 +450,7 @@ const WebApp = ({
         onRefresh={() => setAtTop(false)}
       >
         <WebView
+          autoManageStatusBarEnabled={false}
```
Comment (raymondjacobson) [inline src/components/web/WebApp.tsx:453]: maybe comment what this was for

**Review by raymondjacobson — APPROVED** [review]

File: src/components/web/WebApp.tsx (hunk 2 above)
Comment (piazzatron) [reply src/components/web/WebApp.tsx:453]: good idea

File: package.json (hunk 1 above)
Comment (piazzatron) [reply package.json:52]: 🤞 this fixes you know what

Comment (raymondjacobson) [reply package.json:52]: who shall not be named

**Pull request merged**